### PR TITLE
Add readme information for Vue3

### PR DESCRIPTION
**Pull request by patrick-tolosa**
### Brief Summary of Changes
- Updated the readme to support include Vue3 information
- Restructured the file to make it easier to find what you need.
- Improved the indentations and layout

The new readme can be seen here:  
https://github.com/cloudinary/cloudinary-vue/blob/feature/update-readme-to-include-vue3/README.md

The original one here:
https://github.com/cloudinary/cloudinary-vue/blob/master/README.md

#### What does this PR address?
- [ ] Github issue (Add reference - #XX)
- [x] Refactoring
- [ ] New feature
- [ ] Bug fix
- [ ] Adds more tests

#### Are tests included?  
- [ ] Yes
- [x] No N/A

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,100 +2,100 @@
...
-
-### General usage
+Cloudinary is a [cloud-based service](https://cloudinary.com/solutions) that provides an end-to-end image and video management solution
+including uploads,storage, manipulations, optimizations and delivery. 
+All your media resources are optimized and delivered through a fast CDN using industry best practices.
+
+Using Cloudinary's Vue.js SDK, you can easily perform smart image and video manipulations 
+using code that integrates seamlessly with your existing Vue.js application, and deliver optimized and responsive media to your users.
+
+This Readme provides basic installation and usage information. For the complete documentation, see the [Vue.js SDK guide](https://cloudinary.com/documentation/vue_integration)_
```
Comment (asisayag2) [inline README.md:12]: Something in the layout changed here. why?

File: README.md [hunk 2]
```diff
@@ -2,100 +2,100 @@
...
+                  CldImage,
+                  CldTransformation
+                }
+            })
+            
+            app.mount('#app');
+        ```
+      **Notes**: By default, if `components` is not passed to the Cloudinary plugin, the plugin will _automatically_ install all available Cloudinary components.
+
+2. **A Local setup** - You can also import the cloudinary components manually in each cof your components.
```
Comment (asisayag2) [inline README.md:63]: cof?

**Comment by mickey-gordon** [comment]
Hard to compare this to previous version because something is off in the layout and everything falls under the diff.
Read the new one, and LGTM.

File: README.md (hunk 1 above)
Comment (patrick-tolosa) [reply README.md:12]: this new 'layout' forces line breaks, the end result looks exactly the same but it's written in a way that makes sense to edit.

**Pull request merged**